src/web/app/pages-instructor/instructor-session-result-page/instructor-session-result-question-view.component.html

```diff
@@ -7,12 +7,15 @@
     </div>
   </div>
   <div class="card-body" *ngIf="question.isTabExpanded">
+    <div class="stats-table">
```

Comment:
To clearly show the DOM hierarchy, the line after this needs to be indented. Similarly for the other `stats-table` div down there. It does add a seemingly unnecessary diff for the PR, but this is a necessary evil.